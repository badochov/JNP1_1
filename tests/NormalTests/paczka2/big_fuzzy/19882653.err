Error in line 2: TGKyIKHpne A450 30,6
Error in line 20: kD8pIjhRCB A450 39,95
Error in line 26: RFRIvp4G A2O0 24,4
Error in line 28: TGKyIKHpne A210 12,9
Error in line 33: dkOiuHjg S612 13,1
Error in line 58: dkOiuHjg A210 59,3
Error in line 60: TGKyIKHpne A450 71,5
Error in line 61: dkOiuHjg A450 92,6
Error in line 32: kD8pIjhRCB A450 7,1
Error in line 81: dkOiuHjg A450 38,4
Error in line 93: RFRIvp4G S683 7,1
Error in line 101: dkOiuHjg A450 85,95
Error in line 83: dkOiuHjg S683 35,5
Error in line 98: A450 A210 49,7
Error in line 105: TGKyIKHpne S683 39,5
Error in line 111: RFRIvp4G A450 38,2
Error in line 108: A450 A450 161,4
Error in line 116: u3iCgL S683 30,5
Error in line 123: kD8pIjhRCB S683 50,1
Error in line 125: SA2zkd1M S683 94,1
Error in line 129: SA2zkd1M A210 68,0
Error in line 136: A450 S612 -2,7
Error in line 133: SA2zkd1M S683 64,4
Error in line 134: kD8pIjhRCB A450 21,5
Error in line 140: RFRIvp4G A210 3,3
Error in line 137: SA2zkd1M S983 89,5
Error in line 132: dkOiuHjg A450 314,9
Error in line 160: dkOiuHjg S612 77,5
Error in line 164: dkOiuHjg S683 82,5
Error in line 154: u3iCgL A450 5,0
Error in line 188: dkOiuHjg A450 93,4
Error in line 196: dkOiuHjg A210 54,6
Error in line 199: RFRIvp4G A210 25,19
Error in line 173: RFRIvp4G A450 208,5
Error in line 211: TGKyIKHpne A210 71,9
Error in line 214: kD8pIjhRCB A450 14,1
Error in line 200: RFRIvp4G A210 81,3
Error in line 220: u3iCgL S683 45,2
Error in line 219: RFRIvp4G S683 86,7
Error in line 228: TGKyIKHpne A450 27,5
Error in line 226: RFRIvp4G A210 35,2
Error in line 245: u3iCgL S612 5,9
