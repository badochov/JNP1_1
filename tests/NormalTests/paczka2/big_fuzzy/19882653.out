S612 25,4
S612 75,7
S612 94,9
A450 S 25,4
dkOiuHjg S 69,5
kD8pIjhRCB S 258,7
u3iCgL A 180,1 S 41,0
A210 180,1
S612 94,9
S683 299,7
kD8pIjhRCB S 258,7
dkOiuHjg S 69,5
A210 244,1
dkOiuHjg S 146,1
A450 A 38,8 S 25,4
A450 103,5
u3iCgL A 244,8 S 41,0
A450 A 223,0 S 25,4
RFRIvp4G A 144,3 S 16,9
SA2zkd1M A 40,8 S 7,6
TGKyIKHpne A 101,2 S 117,9
dkOiuHjg A 90,6 S 252,0
kD8pIjhRCB S 258,7
u3iCgL A 244,8 S 41,0
A210 446,4
A450 398,3
S612 239,9
S683 479,6
A210 446,4
dkOiuHjg A 90,6 S 252,0
A210 446,4
A450 A 223,0 S 25,4
RFRIvp4G A 144,3 S 16,9
SA2zkd1M A 301,6 S 7,6
TGKyIKHpne A 101,2 S 117,9
dkOiuHjg A 90,6 S 252,0
kD8pIjhRCB A 0,8 S 258,7
u3iCgL A 252,5 S 41,0
A210 715,7
A450 398,3
S612 239,9
S683 479,6
A450 A 223,0 S 25,4
RFRIvp4G A 144,3 S 16,9
SA2zkd1M A 301,6 S 7,6
TGKyIKHpne A 101,2 S 117,9
dkOiuHjg A 90,6 S 252,0
kD8pIjhRCB A 0,8 S 258,7
u3iCgL A 252,5 S 41,0
A210 715,7
A450 398,3
S612 239,9
S683 479,6
A450 A 223,0 S 25,4
A450 398,3
A450 A 223,0 S 25,4
A450 398,3
kD8pIjhRCB A 96,0 S 258,7
A450 A 223,0 S 25,4
RFRIvp4G A 410,0 S 16,9
SA2zkd1M A 301,6 S 7,6
TGKyIKHpne A 101,2 S 117,9
dkOiuHjg A 90,6 S 252,0
kD8pIjhRCB A 96,0 S 258,7
u3iCgL A 252,5 S 41,0
A210 1076,6
A450 398,3
S612 239,9
S683 479,6
A450 A 223,0 S 25,4
RFRIvp4G A 410,0 S 36,2
SA2zkd1M A 301,6 S 7,6
TGKyIKHpne A 101,2 S 138,6
dkOiuHjg A 90,6 S 252,0
kD8pIjhRCB A 96,0 S 258,7
u3iCgL A 252,5 S 41,0
A210 1076,6
A450 398,3
S612 259,2
S683 500,3
A450 A 223,0 S 25,4
A450 398,3
u3iCgL A 252,5 S 41,0
kD8pIjhRCB A 96,0 S 258,7
A450 A 223,0 S 57,6
RFRIvp4G A 420,1 S 305,3
SA2zkd1M A 301,6 S 7,6
TGKyIKHpne A 204,8 S 138,6
dkOiuHjg A 169,4 S 252,0
kD8pIjhRCB A 126,8 S 258,7
u3iCgL A 252,5 S 52,7
A210 1107,4
A450 590,8
S612 302,7
S683 769,8
RFRIvp4G A 420,1 S 305,3
u3iCgL A 254,0 S 52,7
RFRIvp4G A 451,1 S 305,3
A210 1453,1
kD8pIjhRCB A 126,8 S 313,2
A210 1629,8
dkOiuHjg A 204,5 S 271,0
A450 A 248,6 S 57,6
RFRIvp4G A 500,2 S 305,3
SA2zkd1M A 647,3 S 75,3
TGKyIKHpne A 204,8 S 188,5
dkOiuHjg A 204,5 S 271,0
kD8pIjhRCB A 268,4 S 313,2
u3iCgL A 254,0 S 52,7
A210 1629,8
A450 698,0
S612 420,3
S683 843,3
A450 A 248,6 S 57,6
RFRIvp4G A 500,2 S 305,3
SA2zkd1M A 697,2 S 75,3
TGKyIKHpne A 204,8 S 188,5
dkOiuHjg A 204,5 S 271,0
kD8pIjhRCB A 360,0 S 313,2
u3iCgL A 254,0 S 52,7
A210 1629,8
A450 839,5
S612 420,3
S683 843,3
RFRIvp4G A 500,2 S 305,3
S683 843,3
kD8pIjhRCB A 360,0 S 392,2
S683 932,7
A450 A 330,6 S 57,6
RFRIvp4G A 500,2 S 305,3
SA2zkd1M A 697,2 S 122,3
TGKyIKHpne A 281,6 S 230,9
dkOiuHjg A 593,5 S 342,5
kD8pIjhRCB A 395,5 S 392,2
u3iCgL A 299,6 S 633,4
A210 1783,7
A450 1314,5
S612 1151,5
S683 932,7
S683 934,9
